pkg/controller/nodeipam/ipam/multi_cidr_range_allocator.go

```diff
@@ -117,7 +117,7 @@ func NewMultiCIDRRangeAllocator(
 	testCIDRMap map[string][]*cidrset.ClusterCIDR,
 ) (CIDRAllocator, error) {
 	if client == nil {
-		klog.Fatalf("client is nil")
+		return nil, errors.New("kubeClient is nil when starting multi CIDRRangeAllocator")
```

Comment:
See comment above -- this should not go into a logging change.

Reply:
yeah